### DISCUSSION TITLE
如果搞个网站专门收集西方负面新闻，是不是也可以开个 "The-Run-Philosophy-Organization" 分支？

### DISCUSSION THREAD

**Discussion by zrl4836** [The-Run-Philosophy-Organization/run#198] [General]

**Comment by ghost** [comment]
应该是可以的，写好后也应该可以PR进来
因为这算是避坑（

**Comment by stackunderflow111** [comment]
我们可以加上比如日本研修生，新加坡WP这些，这些很多中介会推荐，但是实际上是和奴工一样的项目，也没办法润，签证一到期就会被赶走没法回去。

我们可以放一些日本研修生被欺负的例子，劝告大家润日本请选择技人国并且请千万不要去研修。也可以放一些新加坡针对WP的奇葩政策（和公民结婚需要批准，女的WP怀孕了会被直接送回国，反正绝对不能让你留下来）。

**Comment by ghost** [comment]
> 我们可以加上比如日本研修生，新加坡WP这些，这些很多中介会推荐，但是实际上是和奴工一样的项目，也没办法润，签证一到期就会被赶走没法回去。
> 
> 我们可以放一些日本研修生被欺负的例子，劝告大家润日本请选择技人国并且请千万不要去研修。也可以放一些新加坡针对WP的奇葩政策（和公民结婚需要批准，女的WP怀孕了会被直接送回国，反正绝对不能让你留下来）。

认同，希望有相关经历的人可以参与编写此项目

**Comment by stackunderflow111** [comment]
> > 我们可以加上比如日本研修生，新加坡WP这些，这些很多中介会推荐，但是实际上是和奴工一样的项目，也没办法润，签证一到期就会被赶走没法回去。
> > 我们可以放一些日本研修生被欺负的例子，劝告大家润日本请选择技人国并且请千万不要去研修。也可以放一些新加坡针对WP的奇葩政策（和公民结婚需要批准，女的WP怀孕了会被直接送回国，反正绝对不能让你留下来）。
> 
> 认同，希望有相关经历的人可以参与编写此项目

我等以后有时间了自己动手写一写。我不觉得申请日本研修生和新加坡WP的人会用github

**Comment by the-run-philosophy** [comment]
认同，可以加入。
